static/js/actions/index.js

```diff
@@ -13,6 +13,7 @@ export const FETCH_FAILURE = 'FETCH_FAILURE';
 export const FETCH_SUCCESS = 'FETCH_SUCCESS';
 export const FETCH_PROCESSING = 'FETCH_PROCESSING';
 
+export const REQUEST_GET_PRICES = 'REQUEST_GET_PRICES';
```

Comment:
Oh gotcha. The action types automatically generated by `redux-hammock` are available on the defined functions, at `actions.prices.get.requestType`, so you should just be able to `import { actions } from '../lib/redux_rest';` and then use that.